planner/core/physical_plans.go

```diff
@@ -74,6 +74,9 @@ type PhysicalTableReader struct {
 	// StoreType indicates table read from which type of store.
 	StoreType kv.StoreType
 
+	// BatchCop = true means the cop task in the physical table reader will be executed in batch mode(use in TiFlash only)
+	BatchCop bool
```

Comment:
It may confuse the user about the [`tidb_allow_batch_cop`](https://docs.pingcap.com/tidb/stable/system-variables#tidb_allow_batch_cop-new-in-v40-version)  and [`max-batch-size`](https://docs.pingcap.com/tidb/stable/tidb-configuration-file#max-batch-size) in tikv-client. 

IIUC, `tidb_allow_batch_cop` merges all the coprocessor tasks to one coprocessor task and sends it to the target tiflash instance. While `max-batch-size` sends several coprocessor tasks in one RPC call between tikv-client and tikv-server.

Since we are going to support `tidb_allow_batch_cop` for tikv instance (https://github.com/pingcap/tidb/issues/19381), the variable name and feature name may further confuse users when they are both supported in TiKV.

Reply:
Yes, I agree, `tidb_allow_batch_cop` is not a good name but it is already released and as far as I know there are already some customers who use this feature in their production env, it does not sound a good idea to change the variable name, so any suggestions to improve this?

Reply:
No idea at present. I think the solution should be backward compatible. Maybe a solution is to add a name alias for `tidb_allow_batch_cop` and recommend users to use that name alias instead.

Reply:
Adding an alias for `tidb_allow_batch_cop` may make user more confused, for example: `tidb_allow_super_cop` is added as an alias for `tidb_allow_batch_cop`, in order to backward compatible, `tidb_allow_batch_cop` must be kept, so the user will find that `set @@tidb_allow_batch_cop = xx` will be overwritten by `set @@tidb_allow_super_cop = xx`, we still need more time/document to explain the relation between `tidb_allow_batch_cop` and `tidb_allow_super_cop`